src/DataLib/fendl32B_retrofit/fendl2_comparison/fendl_comparison.py

```diff
@@ -0,0 +1,289 @@
+   116790.000, 122770.000, 129070.000, 135690.000, 142640.000, 149960.000,
+   157640.000, 165730.000, 174220.000, 183160.000, 192550.000, 202420.000,
+   212800.000, 223710.000, 235180.000, 247240.000, 273240.000, 287250.000,
+   294520.000, 297200.000, 298500.000, 301970.000, 333730.000, 368830.000,
+   387740.000, 407620.000, 450490.000, 497870.000, 523400.000, 550230.000,
+   578440.000, 608100.000, 639280.000, 672060.000, 706510.000, 742740.000,
+   780820.000, 820850.000, 862940.000, 907180.000, 961640.000, 1002600.00,
+   1108000.00, 1164800.00, 1224600.00, 1287300.00, 1353400.00, 1422700.00,
+   1495700.00, 1572400.00, 1653000.00, 1737700.00, 1826800.00, 1920500.00,
+   2019000.00, 2122500.00, 2231300.00, 2306900.00, 2345700.00, 2365300.00,
+   2385200.00, 2466000.00, 2592400.00, 2725300.00, 2865000.00, 3011900.00,
+   3166400.00, 3328700.00, 3678800.00, 4065700.00, 4493300.00, 4723700.00,
+   4965900.00, 5220500.00, 5488100.00, 5769500.00, 6065300.00, 6376300.00,
+   6592400.00, 6703200.00, 7046900.00, 7408200.00, 7788000.00, 8187300.00,
+   8607100.00, 9048400.00, 9512300.00, 10000000.0, 10513000.0, 11052000.0,
+   11618000.0, 12214000.0, 12523000.0, 12840000.0, 13499000.0, 13840000.0,
+   14191000.0, 14550000.0, 14918000.0, 15683000.0, 16487000.0, 16905000.0,
+   17333000.0, 19640000.0
+][::-1]) / 1e6
+
+def args():
+   """"
+   Configure argparser for the FENDL3.2b/FENDL2.0 comparison script.
+
+   Arguments:
+      None
+
+   Returns:
+      argparse.Namespace: Argparse object that contains the user specified
+         arguments for executing the script.
+   """
+
+   parser = argparse.ArgumentParser()
+
+   parser.add_argument(
+      '--element', '-e', required=True, nargs='?',
+      help='''Chemical symbol for the selected element to analyze.'''
+   )
+   parser.add_argument(
+      '--mass_number', '-a', required=True, nargs='?',
+      help = '''Mass number for the selected isotope. If the target is an
+      isomer, type "m" after the mass number in the same string.'''
+   )
+   parser.add_argument(
+      '--emitted_particles', '-p', required=True, nargs='?',
+      help='''String of the emission from the target reaction.
+      For example, for an (n,γ) reaction, type "gamma".'''
+   )
+   
+   return parser.parse_args()
+
+def prepend_with_zeros(cross_sections):
+   """
+   Include zeroes into the beginning of a list such that the total length of
+      of the list is 175, corresponding to the Vitamin-J group structure.
+   
+   Arguments:
+      cross_sections (list): List of cross sections, of length less than or
+         equal to 175.
+   
+   Returns:
+      cross_sections (list): Potentially modified list of cross sections, with
+         zeroes prepended to bring the length of the list to 175.
+   """
+   
+   current_length = len(cross_sections)
+   zeros_needed = 175 - current_length
+   cross_sections = [0] * zeros_needed + cross_sections
+
+   return cross_sections
```

Comment:
I think this should append:
```suggestion
def append_with_zeros(cross_sections):
   """
   Include zeroes into the end of a list such that the total length of
      of the list is 175, corresponding to the Vitamin-J group structure.
   
   Arguments:
      cross_sections (list): List of cross sections, of length less than or
         equal to 175.
   
   Returns:
      cross_sections (list): Potentially modified list of cross sections, with
         zeroes appended to bring the length of the list to 175.
   """
   
   current_length = len(cross_sections)
   zeros_needed = 175 - current_length
   cross_sections = cross_sections + [0] * zeros_needed

   return cross_sections
```